src/main/java/fr/foodlimit/api/shared/models/Place.java

```diff
@@ -0,0 +1,35 @@
+package fr.foodlimit.api.shared.models;
+
+
+import com.fasterxml.jackson.annotation.JsonFormat;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.fasterxml.jackson.annotation.JsonFormat'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)